plugins/vpp/l2plugin/vppcalls/dump_vppcalls.go

```diff
@@ -153,7 +153,11 @@ func (h *BridgeDomainVppHandler) dumpBridgeDomainMacTable() (map[uint32][]*l2nb.
 		} else {
 			arpEntry.IpAddress = ipAddr[:4].To4().String()
 		}
-		arpEntry.PhysAddress = net.HardwareAddr(msg.MacAddress).String()
+		macByte := make([]byte, 6)
```

Comment:
Similar situation here. You can just directly initialize it like this: `PhysAddress = net.HardwareAddr(msg.MacAddress[:])`.